#input
int64 location
---
#output
bool success
